paddle/operators/mul_op.cc

```diff
@@ -47,6 +56,23 @@ class MulOpMaker : public framework::OpProtoAndCheckerMaker {
     AddInput("X", "The first input of mul op");
     AddInput("Y", "The second input of mul op");
     AddOutput("Out", "The output of mul op");
+    AddAttr<int>(
```

Comment:
There is a very useful syntax in `C++ 11`.

```cpp
AddAttr<int>("x_num_col_dims", R"DOC(mul_op can take ...
....
)DOC");
```

`R"LABEL(...)LABEL"`  is just like python's `"""..."""`. which `LABEL` is a custom label to identify where the string begins and ends.

See http://en.cppreference.com/w/cpp/language/string_literal

Reply:
Got it, Thank you!